### DISCUSSION TITLE
Where can I find ms-font-su for Text component

### DISCUSSION THREAD

**Discussion by ademkikaj** [microsoft/fluentui#18516] [Q&A]
I am trying to reach something like this in React environment:

`<span class="ms-font-su ms-fontColor-themePrimary">Big blue text</span>`

Is it a good practice to mix Fabric Core with FluentUI? In the FluentUI is noted that:

_If you're using Fluent UI React, you may not need Fabric Core. Most of the styles from Fabric Core are also available via CSS-in-JS styling, which is the recommended styling approach with Fluent UI React._

Currently, I am using: `<Text variant={"large"} style={{ color:SharedColor.cyanBlue10}}>Big Blue Text</Text>` but I am not getting the same results as the above example that uses FabricCore.

**Comment by micahgodbolt** [comment]
Can you create a codepen with a demo of this issue? aka.ms/fluetuipen
